supplementary/devices/cedric.rst

```diff
@@ -49,13 +58,23 @@ Distributions
    * - Unofficial LineageOS 14.1 (GitHub User: Wzedlare)
      - `android_device_motorola_cedric/cm-14.1 <https://github.com/Wzedlare/android_device_motorola_cedric/tree/cm-14.1>`_
      - `android_kernel_motorola_msm8937/cm-14.1 <https://github.com/Wzedlare/android_kernel_motorola_msm8937/tree/cm-14.1>`_ based on v3.18.49
-     - The vast majority of things
-     - ?
-   * - Unofficial LineageOS 15.1 (GitHub User: Wzedlare)
-     - `android_device_motorola_cedric/lineage-15.1 <https://github.com/Wzedlare/android_device_motorola_cedric/tree/lineage-15.1>`_
-     - `android_kernel_motorola_msm8937/lineage-15.1 <https://github.com/Wzedlare/android_kernel_motorola_msm8937/tree/lineage-15.1>`_ based on v3.18.100
-     - ?
-     - ?
+     - Eveerything
+     - Nothing
+   * - Unofficial LineageOS 15.1 (GitHub User: Sohamlad7)
+     - `android_device_motorola_cedric/lineage-15.1 <https://github.com/Sohamlad7/android_device_motorola_cedric/tree/lineage-15.1>`_
+     - `android_kernel_motorola_msm8937/lineage-15.1 <https://github.com/Sohamlad7/android_kernel_motorola_msm8937/tree/lineage-15.1>`_ based on v3.18.100
+     - Almost everything
+     - USB tethering (can be fixed tho)
+   * - Unoffical LineageOS 16.0 (GitHub User: Sohamlad7)
+     - `android_device_motorola_cedric/lineage-16.0 <https://github.com/Sohamlad7/android_device_motorola_cedric/tree/lineage-16.0>`_
+     - `android_kernel_motorola_msm8937/lineage-16.0 <https://github.com/Sohamlad7/android_kernel_motorola_msm8937/tree/lineage-16.0>`_ based on the newest 3.18
+     - Almost everything
+     - Audio for Wfd & Camera problems
```

Comment:
What is Wfd?

Reply:
Chromecast & Miracast.
Wfd is an abbreviation for Wireless Display.